pandas/tests/computation/test_eval.py

```diff
@@ -1921,7 +1921,7 @@ def test_invalid_parser():
 }
 
 
-@pytest.mark.parametrize("engine", _engines)
+@pytest.mark.parametrize("engine", ENGINES)
 @pytest.mark.parametrize("parser", _parsers)
```

Comment:
should be PARSERS

Reply:
in this case _parsers is a defined a few lines up